custom/enikshay/nikshay_datamigration/management/commands/create_enikshay_cases.py

```diff
@@ -0,0 +1,170 @@
+from django.core.management import BaseCommand
+from django.db import transaction
+
+from casexml.apps.case.const import CASE_INDEX_EXTENSION
+from casexml.apps.case.mock import CaseFactory, CaseStructure, CaseIndex
+from corehq.apps.locations.models import SQLLocation
+
+from custom.enikshay.nikshay_datamigration.models import PatientDetail, Outcome, Followup
+from dimagi.utils.decorators.memoized import memoized
+
+ENIKSHAY_DOMAIN = 'enikshay-np'
```

Comment:
let's make this an argument to `EnikshayCaseFactory` and the management command so we can run this on test domain setups